components/telemetry-operator/apis/telemetry/v1alpha1/logpipeline_types.go

```diff
@@ -93,9 +93,43 @@ type TLSConfig struct {
 
 // Output describes a Fluent Bit output configuration section
 type Output struct {
-	Custom string     `json:"custom,omitempty"`
-	HTTP   HTTPOutput `json:"http,omitempty"`
-	Loki   LokiOutput `json:"grafana-loki,omitempty"`
+	Custom string      `json:"custom,omitempty"`
```

Comment:
I guess since we intend to keep PR smaller, making this only about output fits, but we should also have a look at the rest of the CRD :)